### PR TITLE
Add prettier standard guidelines

### PR DESCRIPTION
**Pull request by Izzur**
This PR related with #13  
I have add standard .prettierrc to set default tab size, also set to auto format on save in vscode

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "printWidth": 80,
+  "tabWidth": 2,
+  "semi": true,
+  "singleQuote": false,
+  "trailingComma": "all",
+  "bracketSpacing": true,
+  "proseWrap": "always",
+  "endOfLine": "auto"
```
Comment (mukarramali) [inline .prettierrc:9]: use `lf` for endOfLine, following Airbnb style guides

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "printWidth": 80,
+  "tabWidth": 2,
+  "semi": true,
+  "singleQuote": false,
```
Comment (mukarramali) [inline .prettierrc:5]: singleQuote as true, please refer to https://github.com/airbnb/javascript#strings--quotes

**Review by mukarramali — CHANGES_REQUESTED** [review]

**Comment by mukarramali** [comment]
Please link issues you are working on

**Comment by Izzur** [comment]
I update the .prettierrc as requested 
This PR related with issue #13

**Review by mukarramali — APPROVED** [review]

**Pull request merged**